### DISCUSSION TITLE
Public and private links displayed (apart from being only copied to clipboard)

### DISCUSSION THREAD

**Discussion by econo202** [mealie-recipes/mealie#1904] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**

Feature request based on the discussion of issue #1880.

When creating a private link (or when one wants to get the public one) it gets copied to clipboard only, and not displayed anywhere. The links are designed to be automatically copied to clipboard, which fails on non-HTTPS setups, but it's impossible to see them to copy them manually.

**Suggest A Solution**

The solution would be to show these links somewhere after they're generated.

**Additional Information**

- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
Public links are no longer a thing, but this is still a valid request for private links
